db/migrate/20170329064703_create_user_emails.rb

```diff
@@ -0,0 +1,31 @@
+class CreateUserEmails < ActiveRecord::Migration
```

Comment:
I would prefer if we can split this migration into the smaller pieces:

1 migration for UserEmails table
1 Migration for transferring records
1 migration for altering the user tables

Smaller migrations are usually recommend so you can easily rollback changes in steps instead of having one giant migration to fight with.